README.md

```diff
@@ -13,7 +13,7 @@
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://github.com/Htetaungkyaw71/Portfolio-Project)
+[Live Demo Link](https://htetaungkyaw71.github.io/Portfolio-Project/)
```

Comment:
- [ ] Consider making sure that your deployment link is working before requesting another review.
![image](https://user-images.githubusercontent.com/52143815/193529552-8a492ccd-f7c6-4918-9cb9-c1d6ad174ccc.png)